### ISSUE TITLE
Cannot see SMB share in android file explorer

### ISSUE THREAD

**Issue by CrazyWolf13** [arpitkh96/AmazeFileManager#4261]
**Describe the bug**
Hi
I used Amaze to mount my NAS to my phone, however after installing Obsidian (Note taking app) and clicking open Vault I get to android file selector, but there I can nowhere see my mounted NAS, am I missing something, or in what path are mounted shares?

**To Reproduce**
1. Open Amaze
2. Mount a SMB share
3. cannot see the share in android file explorer
4. unable to open the obsidian vault, even tough I can 

**Expected behavior**
I expect Amaze to create a physical path, which I can access via other apps.

**Smartphone (please complete the following information):**
 - Device: Samsung Galaxy S22
 - OS: Android 14
 - Rooted: No
 - Version: 3.10

**Additional context**
Thanks for any help!

**Comment by VishnuSanal** [comment]
hi @CrazyWolf13, thanks fro reaching out. can't you see the new location in amaze's drawer (navigation menu)?

**Comment by CrazyWolf13** [comment]
Hi @VishnuSanal 

Thanks for that quick reply!

Yes, I can see it there, but I cannot see it when clicking open File inside Obsidian App, it then opens the android file selector, and there I cannot view the mounted SMB share, does amaze mount it's smb shares to a specific folder?

**Comment by VishnuSanal** [comment]
> Yes, I can see it there, but I cannot see it when clicking open File inside Obsidian App, it then opens the android file selector, and there I cannot view the mounted SMB share, does amaze mount it's smb shares to a specific folder?

hmm 🤔 good catch. let me look into this. btw, @TranceLove related #3953? 🤔

**Comment by CrazyWolf13** [comment]
@VishnuSanal  Yeah right, that could match up, as Obsidian does not even ask with what app to search for a file, it just opens some android like file picker.
